woocommerce/woocommerce
# Allow to display the checkout (page or shortcode) even when the cart is empty

### All Submissions:


### Changes proposed in this Pull Request:

Add woocommerce_checkout_skip_cart_is_empty_check and woocommerce_checkout_skip_update_order_review_expired filters to allow to display the WooCommerce checkout (page or shortcode) even when the cart is empty.

This is useful when using the woocommerce_checkout shortcode on a page where customers can add products to the cart via ajax. I am using this on one of my sites.

Feel free to change the names of the filters to something more appropriate if needs be.

### How to test the changes in this Pull Request:

1. Navigate to the checkout page without any products in the cart.
2. Check that the checkout page appears.

This is similar to the changes made recently in order to display the checkout page in the customizer.

### Other information:


### Changelog entry

Add woocommerce_checkout_skip_cart_is_empty_check and woocommerce_checkout_skip_update_order_review_expired filters to allow to display the WooCommerce checkout (page or shortcode) even when the cart is empty.
